operations/boot_orchestration/Stage_Changes_with_BOS.md

```diff
@@ -0,0 +1,77 @@
+# Staging Changes with BOS
+
+In BOS V2 it is possible to stage changes when creating a session.  These changes will not immediately take effect, and will instead be applied when the `applystaged` endpoint is called.
+
+This is a BOS V2 feature only.  For suggestions on working around this in V1, see [Stage Changes Without BOS](#Stage-Changes-Without-BOS).
+
+## Creating a staged session
+
+Creating a staged session is no different than creating a normal session, with one exception, the `staged` value should be set to `True`.  For more on creating sessions see [Create a New V2 Session](Manage_a_BOS_Session.md#Create-a-New-V2-Session)
+
+(`ncn-mw#`)
+
+```bash
+cray bos v2 sessions create --template-name TEMPLATE_NAME --operation Boot --staged True --format json
+```
+
+This create a new BOS session that can be managed and monitored as normal, but rather than updating the components desired state, the desired state information will be stored in a `staged_state` field.
+The session will continue to run so long as any components have staged state the has not been applied so that status can be used to monitor actions such as rolling upgrades.
+
+## Applying a staged state
+
+Applying staged state is done on a per component basis.  Multiple components can be specified in a single call.
+In the CLI this can be done with a comma separated list of `xnames`, or by specifying a range of values.
+
+(`ncn-mw#`)
+
+```bash
+cray bos v2 applystaged create --xnames x3000c0s19b[1-4]n0
+```
+
+When using the API, components should be provided as a list of xnames.
+(`ncn-mw#`)
+
+```bash
+curl -X POST -H "Authorization: Bearer ${TOKEN}" -H "Content-Type: application/json"  --data '{"xnames":["x3000c0s19b1n0","x3000c0s19b2n0"]}'  x3000c0s19b[1-4]n0 https://api-gw-service-nmn.local/apis/bos/v2/applystaged
+```
+
+When called, any staged data for the given components will be moved to the desired state, clearing out the staged state data.
```

Comment:
In the options section, you should explain the new option that Joel put in about clearing or not clearing the stage.
You'll need to either explain or reference that option here. Note, the default is now to not clear.